server/controllers/VerificationRequestController.ts

```diff
@@ -255,10 +257,15 @@ export class VerificationRequestController {
       const authController = new AuthController();
       await authController.notifyConductorOfVerificationUpdate(foundUser);
 
+      // Generate ADAPT access code if it was requested (and approved)
+      const foundADAPTApp = approvedApps.find((a) => a.get('name') === 'ADAPT');
+      const adaptAccessCode = foundADAPTApp ? await this._generateADAPTAccessCode() : null;
```

Comment:
let's add logic to catch and log any error thrown from `_generateADAPTAccessCode()` and recover so that we don't miss notifying the user. If you want to take it one step further, you can say something like "we weren't able to automatically generate your ADAPT access code, please contact support".